tests/test_slow_async_callbacks.py

```diff
@@ -85,7 +94,21 @@ def test_slow_async_callbacks(exporter: TestExporter) -> None:
                     'code.lineno': 15,
                     'duration': 2.0,
                     'name': 'task bar 1 (bar)',
-                    'logfire.json_schema': '{"type":"object","properties":{"duration":{},"name":{}}}',
+                    'stack': IsJson(
+                        [
+                            {
+                                'code.filepath': 'tests/import_used_for_tests/slow_async_callbacks_example.py',
+                                'code.function': 'bar',
+                                'code.lineno': 15,
+                            },
+                            {
+                                'code.filepath': 'tests/import_used_for_tests/slow_async_callbacks_example.py',
+                                'code.function': 'foo',
+                                'code.lineno': 28,
+                            },
```

Comment:
Agreed per the linear issue that this is somewhat verbose, but I think it's also fine 👍